src/shared-components/Card/ListItems/ListItems.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Box,
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+  Stack,
+  Typography,
+} from '@mui/material';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import { colors } from 'src/styles/colors';
+import { CheckBox } from '@mui/icons-material';
+
+export const ListItems = ({
+  title,
+  description,
+}: {
+  title: string;
+  description: string;
+}) => {
+  return (
+    <List>
+      <ListItem>
```

Comment:
This would be a ListItemSetting component, it should only hold the information need for this particular type and be based on the generic ListItem component that I mentioned above.
Make sure to remove unused imports.

Reply:
Fixed